tests/testthat/_snaps/arg.md

```diff
@@ -1,88 +1,129 @@
+# matches arg
+
+    `arg` must be one of "bar" or "baz", not "foo".
+
+# gives an error with more than one arg
+
+    `arg` must be one of "bar" or "baz", not "bar".
+    i Did you mean "bar"?
+
+# gives error with different than rearranged arg vs value
+
+    `myarg` must be one of "fun" or "bar", not "foo".
+
+---
+
+    `x` must be one of "foo" or "bar", not "foo".
+    i Did you mean "foo"?
+
 # `arg_match()` has informative error messages
 
     Code
-      (expect_error(arg_match0("continuuos", c("discrete", "continuous"), "my_arg")))
+      (expect_error(arg_match_wrapper("continuuos", c("discrete", "continuous"),
+      "my_arg")))
     Output
       <error/rlang_error>
       `my_arg` must be one of "discrete" or "continuous", not "continuuos".
       i Did you mean "continuous"?
+      Context: `arg_match0_wrapper()`
```

Comment:
What's the deal with `Context:`?

Reply:
This shows the `call` condition field. Would you prefer

```
Call: `arg_match0_wrapper()`
```

Reply:
Yeah, call would be better. But why is it displayed here at all?

Reply:
For end users calling `last_error()` we display it for the same reason we display the "in" part in error messages, because it is informative. In unit tests, it's useful to snapshot the "in" part of the messages.

Reply:
Maybe it'd feel more natural with slightly different formatting?

```R
<error/rlang_error>
`my_arg` must be one of "discrete" or "continuous", not "continuuos".
i Did you mean "continuous"?

$call: arg_match0_wrapper()
```

That would make it more obvious that you're displaying a component of the error object, and would be straightforward to generalise in the future.

Reply:
Then should it be `$trace:` as well? The current formatting is meant to be consistent with `Backtrace:`.